apps/settings/locales/settings.en-US.properties

```diff
@@ -1130,8 +1130,9 @@ draw-tile-borders=Draw tile borders
 draw-layer-borders=Draw layer borders
 log-animations=Log slow animations
 remote-debugging-usb=Debugging via USB
-dev-tools-wifi=DevTools via Wi-Fi
-dev-tools-qps=Pseudo-localization
+devtools-wifi=DevTools via Wi-Fi
+devtools-device-name=Device Name
```

Comment:
Hmm, is this a rule for the project? There seems to quite a mix of casing policies in this file, so it certainly doesn't seem obvious that there is "one way". Also, English [title casing rules][1] are typically used for labels like this. Really, I don't care which way it is, but what is the rule and why?

[1]: http://www.grammar-monster.com/lessons/capital_letters_title_case.htm